tests/baselines/reference/enumShadowedInfinityNaN2.js

```diff
@@ -0,0 +1,28 @@
+//// [tests/cases/conformance/enums/enumShadowedInfinityNaN2.ts] ////
+
+//// [enumShadowedInfinityNaN2.ts]
+// repro https://github.com/microsoft/TypeScript/issues/55091
+
+let Infinity = 3;
```

Comment:
Are there any more convincing global names that could get shadowed than Infinity and NaN? It's a stretch for people to create local names `Infinity` or `NaN`.

Reply:
I think those are the only 2 that can be produced by the „constant evaluation”

> It's a stretch for people to create local names Infinity or NaN.

cant disagree :p the issue was labeles as a bug and not a wontfix, it’s similar to the merged: https://github.com/microsoft/TypeScript/pull/55018